cdap-ui/pom.xml

```diff
@@ -314,8 +314,8 @@
                     <copy todir = "${stage.opt.dir}/server/config/themes">
                       <fileset dir = "server/config/themes" />
                     </copy>
-                    <copy todir = "${stage.opt.dir}/graphql">
-                      <fileset dir = "graphql" />
+                    <copy todir = "${stage.opt.dir}/graphql/schema">
```

Comment:
won't you need the fragments file as well?

Reply:
We don't need the fragmentTypes.json in the server right? This gets bundled in the client code in main.js I believe?

Reply:
Correct. fragmentTypes.json is used in the main.js file, but there is a path specified: `import introspectionQueryResultData from '../../graphql/fragments/fragmentTypes.json';`. Is this path still correct after your changes on how to copy the graphql directory?